backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -455,3 +455,103 @@ async def set_memory_provider(provider: str):
+
+        with open(config_path, 'r') as f:
+            full_config = yaml.safe_load(f) or {}
+
+        chat_config = full_config.get('chat', {})
+        system_prompt = chat_config.get('system_prompt', default_prompt)
+
+        # Return just the prompt text, not the YAML structure
+        return system_prompt
+
+    except Exception as e:
+        logger.error(f"Error loading chat config: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)